src/TuDa.CIMS.Web/Components/ShoppingCart/ShoppingCartSubmitPopup.razor.cs

```diff
@@ -28,7 +28,12 @@ public partial class ShoppingCartSubmitPopup
     /// <summary>
     /// The selected working group.
     /// </summary>
-    private WorkingGroup WorkingGroup { get; set; } = null!;
+    public WorkingGroup? WorkingGroup { get; set; } = null;
```

Comment:
🔧 Null is the default value here and can be removed
```suggestion
    public WorkingGroup? WorkingGroup { get; set; }
```